contracts/registry/TrustedIssuersRegistry.sol

```diff
@@ -1,11 +1,11 @@
 pragma solidity ^0.6.0;
 
-
+import "../destroyable/Destroyable.sol";
 import "../registry/ITrustedIssuersRegistry.sol";
 import "@onchain-id/solidity/contracts/IClaimIssuer.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-contract TrustedIssuersRegistry is ITrustedIssuersRegistry, Ownable {
+contract TrustedIssuersRegistry is ITrustedIssuersRegistry, Ownable, Destroyable {
```

Comment:
Inheriting ownable is not required in trusted issuer registry if we are already inheriting it in Destroyable.